### ISSUE TITLE
Button not appearing in toolbar

### ISSUE THREAD

**Issue by i-hardy** [creativeorange/Craft-redactor-font-awesome#5]
Sorry if this is a dumb question, but I'm not seeing any button appear in Redactor fields when I add this plugin to the config. For example, I have a config file like this

```
{
  "buttons": ["html", "formatting", "bold", "italic", "unorderedlist", "link"],
  "plugins": ["fontcolor", "fontAwesome"],
  "fontcolors": ["#ff6861", "#fcba30", "#3e2563"],
  "toolbarFixed": true
}
```

But it doesn't show up any differently to before I added this plugin. Is a button supposed to appear? Should I be adding icons a different way?

I'm running Craft 3.0.17.1 and Redactor 2.1.4. Thanks!

**Comment by jcherniak** [comment]
See #6

**Comment by SanderVanLeeuwen** [comment]
Somehow it's a bit confusing how to actually use the plugin in Craft 3.x as including the 'official' composer package from `svenjungnickel/redactor-font-awesome` does not include the fix from #6. Instead, I had to manually fetch the master branch from *this* repository to get things going.

So, for future reference, add this repository as custom source to your composer.json:
```json
  "repositories": [
    {
      "type": "vcs",
      "url": "https://github.com/creativeorange/Craft-redactor-font-awesome"
    }
  ]
```
and then require `"svenjungnickel/redactor-font-awesome": "dev-master"`.

It would be nice if @svenjungnickel could update his repository to include the fix there as well.

**Comment by svenjungnickel** [comment]
Hey @SanderVanLeeuwen. Tanks for your hint. I updated the repository now. You can find the latest version [here](https://github.com/svenjungnickel/Craft-redactor-font-awesome/releases/tag/2.1.0). You can also install it via composer or with the craft plugin installer in the backend.
Also many thanks to @jcherniak for the changes in the PR.
